runtime/src/main/codestarts/quarkus/quakus-hilla-maven-codestart/codestart.yml

```diff
@@ -0,0 +1,20 @@
+name: quarkus-hilla-maven-codestart
```

Comment:
Here it is singular. Is it meant, that codestarts includes all codestarts, so if others are added later?

Reply:
Exactly. The jar can contain multiple codestarts